packages/core/src/libraries/hook/index.ts

```diff
@@ -188,6 +190,48 @@ export const createHookLibrary = (queries: Queries) => {
     await sendWebhooks(webhooks, consoleLog);
   };
 
+  /**
+   * Trigger data hooks with the given data mutation context. All context objects will be used to trigger hooks.
+   */
+  const triggerExceptionHooks = async (
```

Comment:
Can we refactor and reuse the `triggerDataHooks` method instead of creating a new one? They’re pretty much alike.